docker-compose.yml

```
@@ -43,6 +43,26 @@ services:
       - django
     networks:
       - django
+  redis:
+    image: redis:alpine
+    networks:
+      - django
+  celery:
+    restart: always
+    build:
+      context: ./docker/django
+      dockerfile: Dockerfile
+    command: celery -A tasks worker -l info
+    volumes:
+      - ".:/usr/src/app"
+    env_file:
+     - ./.env
```

```suggestion
     - .env
```
Also I would explicitly specify env variables as in `django` container